supabase/migrations/20260816202033_restrict_app_versions_deleted_at.sql

```
@@ -0,0 +1,76 @@
+-- Upload-scoped principals have app.upload_bundle and can UPDATE non-deleted
+-- app_versions, including deleted_at. Setting deleted_at enqueues storage
+-- deletion. Require bundle.delete for user-context soft-deletes. Internal
+-- service_role paths keep working.
+--
+-- Execution profile (app_versions BEFORE UPDATE OF deleted, deleted_at):
+-- - Where: once per row when deleted_at is set or deleted flips to true.
+-- - Frequency: console-scale bundle deletes (hundreds/day), not plugin hot path.
```

SQLFluff (4.3.0)

[error] Line is too long (81 > 80).

(LT05)